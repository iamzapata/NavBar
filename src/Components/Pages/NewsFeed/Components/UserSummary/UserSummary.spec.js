import React from "react"
import { render, cleanup } from "react-testing-library"
import { Provider } from "react-redux"
import ReduxThunk from "redux-thunk"
import configureMockStore from "redux-mock-store"
import { UserSummary } from "./index"

const createProps = () => ({
  getUserProfile: jest.fn,
  user: {
    first_name: "",
    last_name: "",
    interests: "",
    bio: ""
  }
})

describe("<UserSummary />", () => {
  afterEach(cleanup)
  it("Should render self", () => {
    const { container } = render(<UserSummary {...createProps()} />)
    expect(container.firstChild).toBeTruthy()
    expect(container.firstChild).toMatchSnapshot()
  })
})
