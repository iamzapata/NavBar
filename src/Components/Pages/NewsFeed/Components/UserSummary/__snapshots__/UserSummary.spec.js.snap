// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UserSummary /> Should render self 1`] = `
<div
  class="UserSummary"
>
  <div
    class="card"
  >
    <div
      class="card-content"
    >
      <div
        class="media"
      >
        <div
          class="media-left"
        >
          <figure
            class="image is-48x48"
          >
            <img
              alt="User"
            />
          </figure>
        </div>
        <div
          class="media-content"
        >
          <p
            class="title is-size-4"
          >
             
          </p>
          <p
            class="subtitle is-size-6"
          />
        </div>
      </div>
      <div
        class="content is-size-6"
      />
      <hr />
    </div>
  </div>
</div>
`;
